### PR TITLE
misc: add instructions about test release

### PR DESCRIPTION
**Pull request by lex111**
## Motivation

Reduce the possibility of releasing a "bad" version that will make us hurry and nervously fix it as a hot fix :(

The point is a simple one: before the release of the new version, we build all the packages and try to run the website with them. To make everything as close to reality as possible, we publish the packages in a local private npm registry [Verdaccio](https://github.com/verdaccio/verdaccio) and only then use them for testing purposes.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

See instructions. Tested on Linux, but should work on Mac.

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

File: packages/docusaurus-1.x/lib/core/DocsSidebar.js [hunk 1]
```diff
@@ -8,7 +8,7 @@
 const React = require('react');
 const fs = require('fs');
 const SideNav = require('./nav/SideNav.js');
-const Metadata = require('../core/metadata.js');
+const Metadata = require('./core/metadata.js');
```
Comment (lex111) [inline packages/docusaurus-1.x/lib/core/DocsSidebar.js]: :man_shrugging:

File: packages/docusaurus-plugin-content-blog/src/__tests__/__snapshots__/generateBlogFeed.test.ts.snap [hunk 2]
```diff
@@ -57,7 +57,7 @@ exports[`blogFeed rss can show feed without posts 1`] = `
         <link>https://docusaurus.io/blog</link>
         <description>Hello Blog</description>
         <lastBuildDate>Sun, 25 Oct 2015 23:29:00 GMT</lastBuildDate>
-        <docs>http://blogs.law.harvard.edu/tech/rss</docs>
+        <docs>https://validator.w3.org/feed/docs/rss2.html</docs>
```
Comment (lex111) [inline packages/docusaurus-plugin-content-blog/src/__tests__/__snapshots__/generateBlogFeed.test.ts.snap]: :man_shrugging:

**Comment by lex111** [comment]
It looks like the Verdaccio dependencies in the root workspace affect the rest of the dependencies (including v1). Can we use Docker to run a private proxy?

**Comment by yangshun** [comment]
I don't think we have time to get this working before our next release

**Comment by lex111** [comment]
Sure, it can wait. I think it's best to use Docker here.

**Comment by lex111** [comment]
I updated the instructions, now we only need to execute one command (but Docker installed is required). It is OK?

**Comment by yangshun** [comment]
Yeah sure thing

**Comment by lex111** [comment]
@yangshun ready for review!

**Comment by lex111** [comment]
Of course!

**Comment by lex111** [comment]
I think you can easily googled installation instructions for Docker, especially as this sort information may change over time.

**Comment by yangshun** [comment]
I don't mean the installation instructions. Just mention that we need Docker installed. Because it's not obvious from the instructions unless I read the script.

**Comment by lex111** [comment]
Ah sure, added this to the heading.

**Pull request merged**